### PR TITLE
Add the optional Project content type as a feature

### PR DESCRIPTION
**Pull request by ben-hamelin**
## Summary

Provides the project content type as a feature.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| Documentation reflects changes? | no
| `CHANGELOG` reflects changes? | yes
| Unit/Functional tests reflect changes? | yes
| Did you perform browser testing? | yes
| Risk level | Low
| Relevant links | JIRA issue, bug reports, security alerts, etc.
https://thinkoomph.jira.com/browse/RIG-51

### CODE REVIEW

File: ecms_base/features/custom/ecms_projects/config/install/node.type.project.yml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+dependencies:
+  module:
+    - menu_ui
+third_party_settings:
+  menu_ui:
+    available_menus: {  }
+    parent: ''
+name: Project
+type: project
+description: 'Manage construction projects for DOT '
```
Comment (pfrilling) [inline ecms_base/features/custom/ecms_projects/config/install/node.type.project.yml:12]: Extra space at the end of this description.

**Review by pfrilling — APPROVED** [review]
One trivial fix. Not a blocker by any means.

File: ecms_base/features/custom/ecms_projects/config/install/node.type.project.yml (hunk 1 above)
Comment (ben-hamelin) [reply ecms_base/features/custom/ecms_projects/config/install/node.type.project.yml]: Resolved, and made it generic to "agencies."

**Comment by ben-hamelin** [comment]
@pfrilling ok it's ready!

**Review by pfrilling — APPROVED** [review]
Looks good once the tests pass.

**Pull request merged**